### PR TITLE
Added content proposal details for Contributor Summit EU

### PR DESCRIPTION
**Pull request by geekygirldawn**
Added the details from the content proposal in the Google doc to this page where we have details about the summit in one place. This is the final task required to resolve issue #3273 

/cc @guineveresaenger @jonasrosland

### CODE REVIEW

File: events/2019/05-contributor-summit/README.md [hunk 1]
```diff
@@ -21,10 +21,72 @@ In some sense, the summit is a real-life extension of the community meetings and
...
+
+* 101 New Contributors: audience is brand new contributors who are interested in contributing, but have little to no experience contributing to open source projects.
+* 201 Intermediate New Contributors: audience is people who have experience contributing to other open source projects, but who are new to contributing to k8s.
+
+**Note**: Existing contributors should **not** register for either of these workshops, but you are welcome to attend the social event on Sunday, and you should talk to your SIG chairs to see if they plan to have a SIG F2F meeting on Monday.
+
+### Key Facts:
+
+* We are expecting 200 people with 100 of those attending the 101 New Contributor Workshop and 100 of those attending the 201 Intermediate New Contributor Workshop. 
+* We have space for 140 people in room 4.1, and 100 people in room 5.1.
```
Comment (jonasrosland) [inline events/2019/05-contributor-summit/README.md]: Is this detail needed here?

File: events/2019/05-contributor-summit/README.md [hunk 2]
```diff
@@ -21,10 +21,72 @@ In some sense, the summit is a real-life extension of the community meetings and
...
+* Based on two previous event feedback, folks wanted the following: more hallway track/hang out time, more sessions with actionable items.
+
+### Transparency:
```
Comment (jonasrosland) [inline events/2019/05-contributor-summit/README.md]: Can we move this section down, to keep focus on the Workshop details first?

File: events/2019/05-contributor-summit/README.md (hunk 1 above)
Comment (geekygirldawn) [reply events/2019/05-contributor-summit/README.md]: I wasn't sure, but looking at it again, I think we can delete it.

File: events/2019/05-contributor-summit/README.md (hunk 2 above)
Comment (geekygirldawn) [reply events/2019/05-contributor-summit/README.md]: Sure! I'll add some commits with updates.

**Comment by geekygirldawn** [comment]
@jonasrosland I think I made all of the changes you requested, so this is ready for another review :)

**Comment by jonasrosland** [comment]
/lgtm
👍

**Comment by parispittman** [comment]
/lgtm
/approve

thank you dawn!

**Pull request merged**